docs/evm-sidechain/xrpl-evm-sidechain-faq.md

```diff
@@ -0,0 +1,104 @@
+---
+html: xrpl-evm-sidechain-faq.html
+parent: intro-to-evm-sidechain.html
+blurb: Commonly asked legal questions to consider when running an EVM compatible sidechain.
+labels:
+  - Blockchain
+---
+# XRPL EVM Sidechain FAQ
+
+### Why are Ripple and PeerSyst building an EVM sidechain?
+
+The XRPL natively lacks the full programmability option to add custom business logic. The only programmabilIty option XRPL has today is a “smart transactor” but it's very difficult to write one and any significant change that affects existing protocol needs to go through the amendment process to be approved by the XRPL community. The Hooks feature developed by XRPL Labs promises to bring native programmability to the XRPL. EVM compatible blockchains have “smart contract” functionality that enables developers to build and deploy new applications without a network vote.
+
+EVM compatibility on different blockchains:
+
+| Blockchain Ecosystem | EVM Compatibility Solution/Project |
+|----------------------|------------------------------------|
+| Ethereum             | Native                             |
+| Solana               | Neon                               |
+| Polkadot             | Moonbeam                           |
+| Cosmos               | Evmos                              |
+| Polygon              | zkEVM                              |
+| BNB Chain            | BNB Smart Chain                    |
+| Avalanche            | Avalance C-chain                   |
+| **XRPL**             | **EVM Sidechain**                  |
+
+An EVM sidechain will help the XRPL ecosystem attract EVM developers who can deploy their existing EVM applications on the EVM sidechain with minor modifications.
+
+
+### What are the main benefits of the EVM sidechain?
+
+The main benefits are:
+
+- Faster time to launch by deploying applications as smart contracts.
+- No amendment approval needed to launch applications.
+- Make minor changes to existing smart contracts on other blockchains and launch them on the EVM sidechain.
+- Grow the XRPL ecosystem by adding EVM ecosystem projects and developers.
+
+
+### What are the main risks with the EVM sidechain?
+
+With the XRPL, developers use existing primitives such as payments, DEX, escrow etc. which are secure and have a great track record. With EVM applications, the security is the responsibility of the smart contract developer. Smart contract bugs can get exploited and they have led to a loss of funds.
```

Comment:
.. DEX, and escrows, which are secure and have a great track record.

[no need for _etc_. when you lead with _such as_. Need a comma before _which_ to have the clause apply to the list.]